pkg/sql/opt/optbuilder/groupby.go

```diff
@@ -108,6 +119,16 @@ func (a *aggregateInfo) TypeCheck(ctx *tree.SemaContext, desired *types.T) (tree
 	return a, nil
 }
 
+// IsCommutative checks whether the aggregate is commutative or ordering insensitive.
+func (a aggregateInfo) IsCommutative() bool {
```

Comment:
Ideally, we could reuse `AggregateIsOrderingSensitive` here so we don't have multiple places where we list commutative aggregates.